src/main/scala/fm/sbt/S3URLHandler.scala

```diff
@@ -155,39 +158,39 @@ object S3URLHandler {
     }
   }
 
-  private class BucketSpecificRoleBasedSystemPropertiesCredentialsProvider(providerChain: AWSCredentialsProviderChain, bucket: String)
+  private class BucketSpecificRoleBasedSystemPropertiesCredentialsProvider(providerChain: AwsCredentialsProviderChain, bucket: String)
       extends RoleBasedSystemPropertiesCredentialsProvider(providerChain) {
 
     override val RoleArnKeyNames: Seq[String] = Seq(s"${RoleArnKeyName}.${bucket}", s"${bucket}.${RoleArnKeyName}")
   }
 
-  private class BucketSpecificRoleBasedEnvironmentVariableCredentialsProvider(providerChain: AWSCredentialsProviderChain, bucket: String)
+  private class BucketSpecificRoleBasedEnvironmentVariableCredentialsProvider(providerChain: AwsCredentialsProviderChain, bucket: String)
       extends RoleBasedEnvironmentVariableCredentialsProvider(providerChain) {
 
     override val RoleArnKeyNames: Seq[String] = Seq(s"${RoleArnKeyName}.${bucket}", s"${bucket}.${RoleArnKeyName}")
   }
   
   private def toEnvironmentVariableName(s: String): String = s.toUpperCase.replace('-','_').replace('.','_').replaceAll("[^A-Z0-9_]", "")
 
-  private def makePropertiesFileCredentialsProvider(fileName: String): PropertiesFileCredentialsProvider = {
-    val file: File = new File(DOT_SBT_DIR, fileName)
-    new PropertiesFileCredentialsProvider(file.toString)
-  }
+//  private def makePropertiesFileCredentialsProvider(fileName: String): PropertiesFileCredentialsProvider = {
+//    val file: File = new File(DOT_SBT_DIR, fileName)
+//    new PropertiesFileCredentialsProvider(file.toString)
+//  }
```

Comment:
PropertiesFileCredentialsProvider was removed in v2... https://github.com/aws/aws-sdk-java-v2/blob/master/docs/LaunchChangelog.md